spec/symbol_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'stock_quote/symbol'
+require 'stock_quote/utility'
+require 'spec_helper'
+
+describe StockQuote::Symbol  do
+  describe 'symbol_lookup' do
+    context 'success' do
+      describe 'apple company', vcr: { cassette_name: 'apple_lookup'} do
+        @fields = StockQuote::Symbol::FIELDS
+
+        @fields.each do |field|
+          it ".#{to_underscore(field)}" do
+            symbol = StockQuote::Symbol.symbol_lookup('apple').first
+            expect(symbol).to respond_to(to_underscore(field).to_sym)
+          end
+
+          it ".#{field}" do
+            symbol = StockQuote::Symbol.symbol_lookup('apple').first
+            expect(symbol).to respond_to(field.to_sym)
+          end
+        end
+
+        it 'returns sucessful result for query with company name' do
+          symbols = StockQuote::Symbol.symbol_lookup('apple')
+          expect(symbols.count).to eq(10)
+          expect(symbols.map(&:exch).uniq.sort).to eq(["EBS", "FRA", "GER", "LSE", "NAS", "NYQ", "PNK", "SHH"])
```

Comment:
I am getting an error on this spec:

```
expected: ["EBS", "FRA", "GER", "LSE", "NAS", "NYQ", "PNK", "SHH"]
got: ["EBS", "GER", "ISE", "LSE", "NAS", "NYQ", "PNK", "SHH"]
```
How about we modify this to test for the inclusion of the symbol 'AAPL' instead, since we know that will always be included (for as long as Apple exists).

```
expect(symbols.map(&:symbol).uniq).to include("AAPL")
```